client/src/app/submission/submission.component.ts

```diff
@@ -94,6 +102,24 @@ export class SubmissionComponent implements OnInit, OnDestroy {
     });
   }
 
+  getValidationSummary() {
+    this.validationErrors = {
+      biomaterials: {
+        total: 4,
+        metadataErrors: 4,
+      },
+      processes: {
+        total: 4,
+        metadataErrors: 4
+      },
+      files: {
+        total: 24,
+        metadataErrors: 4,
+        fileMissing: 20
+      }
+    };
+  }
+
```

Comment:
could define an interface for validation summary

Reply:
Will be done in next PR